include/picongpu/plugins/openPMD/openPMDWriter.hpp

```diff
@@ -496,7 +510,7 @@ make sure that environment variable OPENPMD_BP_BACKEND is not set to ADIOS1.
 
                 HDINLINE void operator()(ThreadParams* params)
                 {
-#ifndef __CUDA_ARCH__
+#    ifndef __CUDA_ARCH__
```

Comment:
clang-format is being weird with those

Reply:
There is a clang-format issue in CI (still?). Could it be that you are using a different version from our CI and so this part gets treated differently.

Reply:
clang-format on my machine was in a weird mood yesterday, now it's doing this correctly again